### DISCUSSION TITLE
[Request] Entity hider - hide clan chat members and friends 2D

### DISCUSSION THREAD

**Discussion by pogof** [runelite/runelite#18300] [Ideas]
As title says. Right now you can "hide NPCs 2D, Local players 2D, Other player 2D", however when someone becomes clan chat member or your friend "Hide Other Player 2D" doesnt work on them anymore, forcing you either remove the people from friends, leave clan chat or hiding the whole character with "Hide friend" which is not ideal in certain cases.

**Comment by ike2117** [comment]
Currently experiencing this too. Playing on fixed size leaves me with limited screen realestate so I like to keep only whats necessary. I play with hide other player 2d but now after joining a clan and starting to raid, its getting in the way. I'd love if more options were added to hide more 2d options

**Comment by MarniRS** [comment]
I support more options for Hide 2D's aswell. Ever since 'Hide party members' got added as a config to the plugin i can no longer hide the 2D's of accounts in my party and hiding them completely is not ideal.

**Comment by DesignSir** [comment]
Support this idea - 2d's in 8-man TOAs can make it very hard to see whats going on and it feels weird to be punished for having people on my friends list

turning hide friends on and off while raiding is a work around, but i like to see where they are stood to avoid boulders etc and its a bit annoying to turn on and off
